tests/PHPStan/Rules/Variables/IssetRuleTest.php

```diff
@@ -281,7 +281,7 @@ public function testVariableCertaintyInIsset(): void
 				116,
 			],
 			[
-				'Variable $variableInSecondCase in isset() always exists and is always null.',
+				'Variable $variableInSecondCase in isset() always exists and is not nullable.',
```

Comment:
this one is actually interessting, as I think it actually fixes a bug - which is suprising.

https://github.com/phpstan/phpstan-src/blob/3a5328b15858b1e484b011bfc61075655cc3574b/tests/PHPStan/Rules/Variables/data/variable-certainty-isset.php#L117

Reply:
having another look at the test-case I think our assertions could be more precise:

https://github.com/phpstan/phpstan-src/blob/3a5328b15858b1e484b011bfc61075655cc3574b/tests/PHPStan/Rules/Variables/data/variable-certainty-isset.php#L108-L124

in case 1 and 2 the case expression is always true and we are only assigning values. as the switch-case is constructed we will always run into `break` on line 114, which effectively means the cases below line 114 are dead code.

Reply:
found the reason for the change in behaviour:

$neverType->isNull() -> no
$nullType->isSuperTypeOf($neverType) -> yes